src/EOS.hpp

```diff
@@ -84,8 +84,14 @@ AMREX_FORCE_INLINE AMREX_GPU_HOST_DEVICE auto EOS<problem_t>::ComputeTgasFromEin
 	Tgas = chemstate.T;
 #else
 	if constexpr (gamma_ != 1.0) {
-		const amrex::Real c_v = boltzmann_constant_ / (mean_molecular_weight_ * (gamma_ - 1.0));
-		Tgas = Eint / (rho * c_v);
+		// const amrex::Real c_v = boltzmann_constant_ / (mean_molecular_weight_ * (gamma_ - 1.0));
+		// Tgas = Eint / (rho * c_v);
+		burn_t chemstate;
```

Comment:
In line 89 above, instead of using the `burn_t` object, we should now be able to use the `chem_eos_t` struct that Max created in https://github.com/AMReX-Astro/Microphysics/pull/1199

But, I get an error if I use this struct in Quokka. I include eos.H above, which in turn includes eos_type.H which is where this new struct is defined. 
@BenWibking